sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_create_client_async.py

```diff
@@ -27,53 +27,40 @@
 class CreateClients(object):
     def __init__(self):
         load_dotenv(find_dotenv())
-        self.account_url = os.environ["CONTAINERREGISTRY_ENDPOINT"]
 
     async def create_registry_client(self):
         # Instantiate the ContainerRegistryClient
         # [START create_registry_client]
         from azure.containerregistry.aio import ContainerRegistryClient
         from azure.identity.aio import DefaultAzureCredential
+        account_url = os.environ["CONTAINERREGISTRY_ENDPOINT"]
 
-        client = ContainerRegistryClient(self.account_url, DefaultAzureCredential())
+        client = ContainerRegistryClient(account_url, DefaultAzureCredential())
         # [END create_registry_client]
 
-    async def create_repository_client(self):
```

Comment:
did you move the creating repository client sample elsewhere?

Reply:
This object isn't meant to be instantiated directly so I removed the sample